test/unit/LedgerId.js

```diff
@@ -0,0 +1,117 @@
+        expect(LedgerId.fromString(HEX_PREVIEWNET).isTestnet()).to.eql(false);
+        expect(LedgerId.fromString(HEX_PREVIEWNET).isPreviewnet()).to.eql(true);
+
+        try {
+            LedgerId.fromString(OTHER);
+        } catch (error) {
+            expect(error.message).to.eql("Default reached for fromString");
+        }
+
+        expect(LedgerId.fromString(HEX_OTHER).toString()).to.eql("03");
+        expect(LedgerId.fromString(HEX_OTHER).isMainnet()).to.eql(false);
+        expect(LedgerId.fromString(HEX_OTHER).isTestnet()).to.eql(false);
+        expect(LedgerId.fromString(HEX_OTHER).isPreviewnet()).to.eql(false);
+    });
+
+    it("isMainnet|Testnet|Previewnet|Other returns boolean", function () {
+        let ledgerId = LedgerId.fromString("0");
+        expect(ledgerId.isMainnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+        ledgerId = LedgerId.fromString(MAINNET);
+        expect(ledgerId.isMainnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+        ledgerId = LedgerId.fromBytes(new Uint8Array([0]));
+        expect(ledgerId.isMainnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+
+        ledgerId = LedgerId.fromString("1");
+        expect(ledgerId.isTestnet()).to.eql(true);
+        expect(ledgerId.isMainnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+        ledgerId = LedgerId.fromString(TESTNET);
+        expect(ledgerId.isTestnet()).to.eql(true);
+        expect(ledgerId.isMainnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+        ledgerId = LedgerId.fromBytes(new Uint8Array([1]));
+        expect(ledgerId.isTestnet()).to.eql(true);
+        expect(ledgerId.isMainnet()).to.eql(false);
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+
+        ledgerId = LedgerId.fromString("2");
+        expect(ledgerId.isPreviewnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isMainnet()).to.eql(false);
+        ledgerId = LedgerId.fromString(PREVIEWNET);
+        expect(ledgerId.isPreviewnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isMainnet()).to.eql(false);
+        ledgerId = LedgerId.fromBytes(new Uint8Array([2]));
+        expect(ledgerId.isPreviewnet()).to.eql(true);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isMainnet()).to.eql(false);
+
+        ledgerId = LedgerId.fromBytes(new Uint8Array([3]));
+        expect(ledgerId.isPreviewnet()).to.eql(false);
+        expect(ledgerId.isTestnet()).to.eql(false);
+        expect(ledgerId.isMainnet()).to.eql(false);
+    });
+
+    it("toString throws error when length > 1", function () {
```

Comment:
This should not be the case. `LedgerId.toString()` should work just fine if the bytes are longer than 1, but if it is exactly 1 we should check to see if the value is `0`, `1`, or `2` in which case we'd print the the network's name instead of hex: `0` => `"mainnet"`, etc.